.github/workflows/ci-test-helm-charts.yml

```diff
@@ -14,8 +14,11 @@ on:
 
 jobs:
   lint:
-    name: Helm Chart Tests / ubuntu 20.04
-    runs-on: "ubuntu-20.04"
+    name: ${{ matrix.name }}
```

Comment:
@cadedupont this is how we should be doing it.
https://github.com/kubearmor/KubeArmor/blob/main/.github/workflows/ci-test-ginkgo.yml#L21:L29

@daemon1024 are suggesting that we add multiple Ubuntu versions to matrix?

Reply:
We keep single version runner in Matrix, but it may be such that we add self hosted runners to test multiple runners so let's keep the actions to leverage Matrix for now.